src/content/docs/logs/get-started/new-relics-log-management-security-privacy.mdx

```diff
@@ -34,12 +45,8 @@ Numbers that appear to be a credit card (thirteen to sixteen digits) are obfusca
 
 Nine-digit numbers with hyphens that appear to be Social Security numbers, such as `123-45-6789`, are obfuscated as `XXXXXXXXX`. Nine-digit numbers with spaces, such as `123 45 6789`, or hyphens in a different pattern, such as `12-345-67-89`, are not automatically obfuscated.
 
-If you need to opt out of automatic obfuscation, get support at [support.newrelic.com](https://support.newrelic.com).
-
-## Customize your security settings [#customize-security]
-
-The data you send to New Relic, including any additional filtering, is controlled by the configuration of the [log forwarder](/docs/logs/forward-logs/enable-log-management-new-relic/) you use. You control what customer data is logged, so be sure to follow your organization's security guidelines to mask, obfuscate, or prevent sending any sensitive data.
+If you need to opt out of automatic obfuscation, [contact support](https://support.newrelic.com).
```

Comment:
Would it make sense to include this in a `Callout` of type `important`, so it becomes more visible?